ArcTouch.Core/ViewModels/HomeViewModel.cs

```diff
@@ -0,0 +1,162 @@
+﻿using ArcTouch.Core.Models;
+using ArcTouch.Core.Models.Api;
+using MvvmCross.Commands;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcTouch.Core.ViewModels
+{
+    public class HomeViewModel : BaseViewModel<object, object>
+    {
+        private int Page { get; set; } = 1;
+        private Task MoviesTask { get; set; }
```

Comment:
You don't need to define getters and setters for private fields

Reply:
I actually always starts with private. and i modify the property if i see that is necessary. so... i think i got this habit. and if i need to do something to have that value. a logic or something. i don't have to change even inside the class. i don't disagree. just saying what i normally do.

Reply:
I could be wrong tho!